Package.swift

```diff
@@ -176,6 +176,10 @@ let package = Package(
       url: "https://github.com/SlaunchaMan/GCDWebServer.git",
       revision: "935e2736044e71e5341663c3cc9a335ba6867a2b"
     ),
+    .package(
+      url: "https://github.com/GoogleCloudPlatform/recaptcha-enterprise-mobile-sdk.git",
+      "18.2.0" ..< "18.2.1"
```

Comment:
Generally we want to accept bug fix releases of dependencies without having to respin Firebase. Should it be `"18.2.0" ..< "18.3.0"`. Also, this would make it consistent with the `~>` operator in the podspec.